CMakeLists.txt

```diff
@@ -379,7 +399,7 @@ endif()
 if(ENABLE_KVS_THREADPOOL)
   file(GLOB THREADPOOL_SOURCE_FILES "src/source/Threadpool/*.c")
   add_library(kvsWebRtcThreadpool ${LINKAGE} ${THREADPOOL_SOURCE_FILES})
-  target_link_libraries(kvsWebRtcThreadpool PRIVATE kvspicUtils)
+  target_link_libraries(kvsWebRtcThreadpool PUBLIC kvspicUtils)
```

Comment:
All consuming libraries of **kvsWebRtcThreadpool** (A.K.A. `EXTRA_DEPS`) already link with **kvspicUtils**, so seems PRIVATE linking would be more appropriate here since we don not need to expose the consuming libraries to **kvspicUtils** _through_ kvsWebRtcThreadpool. Is there another reason this change was made?